uikit/viewmodels/src/main/java/com/sygic/maps/uikit/viewmodels/searchresultlist/databinding/BindingAdapters.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019 Sygic a.s. All rights reserved.
+ *
+ * This project is licensed under the MIT License.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.sygic.maps.uikit.viewmodels.searchresultlist.databinding
+
+import androidx.databinding.BindingAdapter
+import com.sygic.maps.uikit.views.searchresultlist.SearchResultList
+import com.sygic.maps.uikit.views.searchresultlist.adapter.ResultListAdapter
+import com.sygic.sdk.search.SearchResult
+
+@BindingAdapter(value = ["adapter"])
+fun setAdapter(
+    searchResultList: SearchResultList,
+    adapter: ResultListAdapter<SearchResult, ResultListAdapter.ItemViewHolder<SearchResult>>
+) = searchResultList.setAdapter(adapter)
```

Comment:
nie je toto zbytocne, ak sa attribute vola `adapter` a list ma metodu `setAdapter`? nic ine okrem setteru sa tu nerobi, takze toto by mal binding zvladnut aj bez binding adapteru nie?

Reply:
toto jsme sem dali kvůli těm generikám, se kterými si databinding neporadil, nevzpomínáš?

Reply:
hmm to ma nenapadlo 😃👍